webpack.config.js

```diff
@@ -217,6 +226,10 @@ const testBundle = () => {
 				name: 'Test files',
 				color: '#34a853',
 			} ),
+			new CleanWebpackPlugin( {
+				// Stop this build from removing files created by one of the other builds.
```

Comment:
Ohhh, I get what this means. Let's be a bit more specific here still—I can imagine this puzzling someone later because "build" is still a bit ambiguous here:
```suggestion
				// Prevent this build from removing files created by one of the other builds
				// (eg. Module Entry Points and Plugin CSS).
```